autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py

```diff
@@ -198,15 +198,25 @@ def generate_surfer_reply(
         if (viewport["pageTop"] + viewport["height"] + 5) < viewport["scrollHeight"]:
             text_labels += f"""
   {{ "id": {MARK_ID_PAGE_DOWN}, "aria-role": "scrollbar", "html_tag": "button", "actions": ["click"], "name": "browser scroll down control" }},"""
-
-        # Everything visible
+        
         for r in visible_rects:
             if r in rects:
                 actions = '["click"]'
                 if rects[r]["role"] in ["textbox", "searchbox", "search"]:
                     actions = '["type"]'
-                text_labels += f"""
-   {{ "id": {r}, "aria-role": "{rects[r]['role']}", "html_tag": "{rects[r]['tag_name']}", "actions": "{actions}", "name": "{rects[r]['aria-name']}" }},"""
+                if rects[r]["role"] in ["combobox", "menu", "listbox"]:
```

Comment:
I am not sure if this is a universal way to enumerate options. It's possible we might want to add this the javascript